walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -303,5 +408,27 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void bothOptionalsTooBusy() {
+    // No mandatory attendees, and two optional attendees who are busy all day. // The result should be that there is no possible meeting time.
```

Comment:
Nit: the two comments should be on separate lines

Reply:
Fixed, thanks!